Name Selection Improvements

Jiiniasu opened the pull request:
Fix: Infinite scroll bug with mousewheel
Change: Offset by row instead of page

What do you think?

lukeIam commented:
Got this when switching to the player selection screen:
![image](https://user-images.githubusercontent.com/5115160/33233463-87606d42-d216-11e7-9bb6-72e93e73eec5.png)
![image](https://user-images.githubusercontent.com/5115160/33233478-b998d164-d216-11e7-9ab5-bd028ffa0f4a.png)
(max = -3)

Jiiniasu commented:
@lukeIam Thanks for the feedback. Seems like I forgot to consider a profile count less than the amount of tiles.

Updated the branch and added a fix.
